docs/source/getting_started/notebooks.rst

```diff
@@ -11,6 +11,7 @@ Some of these demonstrate the speed-ups from our work, even with already short t
     `Functional API`_, Use our methods within your own trainer, |colab_functional|_
     `Trainer`_, Getting started with our Composer :class:`.Trainer`, |colab_trainer|_
     `NLP`_, Fine-tune NLP models from |:hugging_face:| Transformers with our trainer, |colab_nlp|_
```

Comment:
Should the old NLP notebook still be here? (I don't know the answer)

Reply:
I think it's nice to show people have to define different composer models and it could serve as a nice example of how to move from using the built in integrations to rolling your own for more customization. That said there is a significant overlap and I see how it could cause confusion. deleting it is probably out of scope for this PR though.